packages/flex-plugin-scripts/src/config/__tests__/devServer/pluginServer.test.ts

```diff
@@ -27,99 +27,41 @@ describe('pluginServer', () => {
     jest.spyOn(fsScript, 'getPaths').mockReturnValue(paths);
   });
 
-  describe('_rebasePlugins', () => {
-    it('should return empty array of localPlugin is empty', () => {
-      const remotePluginOne = { name: 'plugin-remote-1' } as Plugin;
-      const remotePluginTwo = { name: 'plugin-remote-2' } as Plugin;
+  describe('_mergePlugins', () => {
+    it('should return plugins w phase >=3', () => {
```

Comment:
`_mergePlugin` should do nothing with `phase` - the phase should be filtered out in `_getRemotePlugins`. `_mergePlugin` should be agnostic to phase.